src/Core/src/nuget/buildTransitive/Microsoft.Maui.Core.After.targets

```diff
@@ -7,6 +7,32 @@
 
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.Maui.Core.BundledVersions.targets" />
 
+  <!-- 
+    Check to make sure that .NET MAUI packages are not installed as .NET Standard
+    when they are installed into a .NET "Core" project. If this happens, it means
+    that the user has installed a .NET MAUI package that requires a later TFM than
+    what the project has.
+
+    Some examples:
+     - Installing a .NET 9 MAUI package into a .NET 8 project
+     - Installing a .NET 9 MAUI package into a very old .NET 9 installation
+
+    To skip this check, set $(SkipCheckForMauiMinimumSupportedWorkloadVersion) to true.
+  -->
+  <Target
+    Name="_CheckForMauiMinimumSupportedWorkloadVersion"
+    BeforeTargets="_CheckForInvalidConfigurationAndPlatform;_CheckForUnsupportedTargetPlatformIdentifier"
+    Condition="
+      '$(SkipCheckForMauiMinimumSupportedWorkloadVersion)' != 'true' and
+      '$(MauiWorkloadVersion)' != '' and
+      $([MSBuild]::VersionLessThan('$(MauiWorkloadVersion)', '$(MauiMinimumSupportedWorkloadVersion)'))">
+    
+    <Error
+        Code="MA003"
+        Text="The current .NET MAUI package version '$(MauiCorePackageVersion)' requires the .NET MAUI workload version to be at least '$(MauiMinimumSupportedWorkloadVersion)'. The current .NET MAUI workload version is '$(MauiWorkloadVersion)'." />
```

Comment:
I think we need to localize this, now that we have the one loc working back again we might want to create some locale setup for our targets.

Reply:
Maybe we can create a new issue to follow up on this with @tj-devel709

Reply:
Created an issue: https://github.com/dotnet/maui/issues/26308